fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -704,6 +709,18 @@ public class SessionVariable implements Serializable, Writable, Cloneable {
     @VariableMgr.VarAttr(name = ENABLE_SPILL)
     private boolean enableSpill = false;
 
+    @VariableMgr.VarAttr(name = ENABLE_HASH_JOIN_SPILL, flag = VariableMgr.INVISIBLE)
+    private boolean enableHashJoinSpill = true;
+
+    @VariableMgr.VarAttr(name = ENABLE_AGG_SPILL, flag = VariableMgr.INVISIBLE)
+    private boolean enableAggSpill = true;
+
+    @VariableMgr.VarAttr(name = ENABLE_AGG_DISTINCT_SPILL, flag = VariableMgr.INVISIBLE)
+    private boolean enableAggDistinctSpill = true;
+
+    @VariableMgr.VarAttr(name = ENABLE_SORT_SPILL, flag = VariableMgr.INVISIBLE)
+    private boolean enableSortSpill = true;
```

Comment:
how about union them into a mask? per bit stands for different operators.

Reply:
> how about union them into a mask? per bit stands for different operators.

I think this is not intuitive enough for users, so I tend to keep the current way and just use mask on thrift